jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/RangeSlider.java

```diff
@@ -85,34 +101,51 @@ public List<String> getThumbLabelValue() {
                 .collect(Collectors.toList());
     }
 
-    @JDIAction("Set horizontal slider from '{name}' to {0}, {1}")
-    public void slideHorizontalTo(int valueLeft, int valueRight) {
-        double trackWidth = getTrackContainer().getSize().width;
-        double minValue = Double.parseDouble(getThumbContainer().get(1).getAttribute("aria-valuemin"));
-        double maxValue = Double.parseDouble(getThumbContainer().get(1).getAttribute("aria-valuemax"));
-        List<Double> nowValue = Arrays.stream(getThumbContainer().get(1).getAttribute("aria-valuenow").split(","))
-                .map(Double::parseDouble)
-                .collect(Collectors.toList());
-        double pixelsInUnit = trackWidth / (maxValue - minValue);
-        double xOffsetLeft = (valueLeft - nowValue.get(0)) * pixelsInUnit;
-        getLeftThumb().dragAndDropTo((int) Math.round(xOffsetLeft), 0);
-        double xOffsetRight = (valueRight - nowValue.get(1)) * pixelsInUnit;
-        getRightThumb().dragAndDropTo((int) Math.round(xOffsetRight), 0);
-    }
-
-    @JDIAction("Set vertical slider from '{name}' to {0}, {1}")
-    public void slideVerticalTo(int valueLeft, int valueRight) {
-        double trackHeight = getTrackContainer().getSize().height;
-        double minValue = Double.parseDouble(getThumbContainer().get(1).getAttribute("aria-valuemin"));
-        double maxValue = Double.parseDouble(getThumbContainer().get(1).getAttribute("aria-valuemax"));
-        List<Double> nowValue = Arrays.stream(getThumbContainer().get(1).getAttribute("aria-valuenow").split(","))
-                .map(Double::parseDouble)
-                .collect(Collectors.toList());
-        double pixelsInUnit = trackHeight / (maxValue - minValue);
-        double yOffsetLeft = (valueLeft - minValue) * pixelsInUnit - (nowValue.get(0) - minValue) * pixelsInUnit;
-        getLeftThumb().dragAndDropTo(0, -(int) Math.round(yOffsetLeft));
-        double yOffsetRight = (valueRight - minValue) * pixelsInUnit - (nowValue.get(1) - minValue) * pixelsInUnit;
-        getRightThumb().dragAndDropTo(0, -(int) Math.round(yOffsetRight));
+    @JDIAction("Set slider from '{name}' to {0}")
+    public void setLeftValue(int valueLeft, boolean isVertical) {
+        List<Double> nowValue = getCurrentLocationValue();
+        if (isVertical) {
+            double trackHeight = getTrackContainer().getSize().height;
+            double minValue = Double.parseDouble(getThumbContainer().get(1).getAttribute("aria-valuemin"));
+            double pixelsInUnit = getPixelsInUnit(trackHeight);
+            dragAndDropToYOffsetLeft(valueLeft, minValue, pixelsInUnit, nowValue);
+        } else {
+            double trackWidth = getTrackContainer().getSize().width;
+            double pixelsInUnit = getPixelsInUnit(trackWidth);
+            dragAndDropToXOffsetLeft(valueLeft, pixelsInUnit, nowValue);
```

Comment:
dragAndDropToXOffsetLeft() method is using getLeftThumb().dragAndDropTo(). I just moved it to separate method because otherwise is was a code duplication after adding new setLeftValue, setRightValue and setValue methods